src/app/ui/hud/stance.js

```diff
@@ -12,9 +12,10 @@ module.exports = class Stance {
   }
 
   view() {
-    const sess = Session.focused()
-    if (!sess) return
-    const stance = Lens.get(sess, "state.stance.text")
+    const stance = Lens.get(
+      Session.current,
+      "state.stance.text"
+    )
```

Comment:
Is this file needed at all anymore? I see `vital.js` doesn't call it anymore and that seems fine if it can be simplified. Although the UI should probably list like "offensive", "defensive", "guarded" and all that rather than this internal ID.

<img width="285" alt="Screen Shot 2020-08-16 at 2 40 45 PM" src="https://user-images.githubusercontent.com/69156/90344436-a56f1900-dfce-11ea-8434-c6ba1da63604.png">